src/bp/nlu-server/model-service.ts

```diff
@@ -0,0 +1,84 @@
+import { NLU } from 'botpress/sdk'
+import crypto from 'crypto'
+import fse, { WriteStream } from 'fs-extra'
+import _ from 'lodash'
+import path from 'path'
+import { Stream } from 'stream'
+import tar from 'tar'
+import tmp from 'tmp'
+
+export default class ModelService {
+  constructor(private modelDir: string) {}
+
+  public async init() {
+    const modelDirExists = fse.existsSync(this.modelDir)
+    if (!modelDirExists) {
+      await fse.mkdir(this.modelDir)
+    }
+  }
```

Comment:
`mkdirp.sync()` will create the folder if it doesn't exist and will do nothing if it already does, so you don't need your existsync